modules/gcp-sa-provisionning/main.tf

```diff
@@ -22,7 +22,7 @@ resource "google_service_account_iam_policy" "impersonatable" {
 
 
 resource "google_project_iam_custom_role" "workspace_creator" {
-  role_id = "${var.prefix}_workspace_creator"
```

Comment:
Hmmmm, according to the docs we can use `_` characters: https://cloud.google.com/iam/docs/roles-overview#custom-role-creation

Reply:
Most probably probles are in the prefix?